Src/BootCamp.Chapter/Checks.cs

```diff
@@ -16,26 +16,26 @@ public static class Checks
     {
         public static int PromptInt(string message)
         {
-            // To do: call your implementation. 
+            Lesson3.RequestInt(message);
             return 0;
         }
 
         public static string PromptString(string message)
         {
-            // To do: call your implementation. 
+            Lesson3.RequestString(message);
             return "";
         }
 
         public static float PromptFloat(string message)
         {
-            // To do: call your implementation. 
+            Lesson3.RequestFloat(message);
             return 0;
         }
 
         public static float CalculateBmi(float weight, float height)
         {
-            // To do: call your implementation. 
+            Lesson3.CalculateBodyMassIndex(weight, height);
             return 0;
```

Comment:
You want to return the Lesson3.X(Y) stuff and not those default values.

Reply:
Thanks. I think I've gotten it now.